src/style.css

```diff
@@ -0,0 +1,17 @@
+:root {
+  font-family: Red Hat Text;
```

Comment:
Any specific reason behind using Red Hat Text font?

Reply:
I like it

Reply:
Use Open Sans for now.

We'll be changing it later once we finalize the brand standards.